html/navbar.html

```diff
@@ -15,6 +15,9 @@
           </li>
           <li class="nav-item">
             <a class="nav-link" href="/network">Network</a>
+          </li>
+             <li class="nav-item">
+            <a class="nav-link" href="/wifi-networks">Wifi Networks</a>
```

Comment:
Please write it was "WiFi" everywhere that visible to the user. It's a bit inconsistent at the moment.

Reply:
This still needs to be addressed